app/routes/circulars.$circularId.($version)/route.tsx

```diff
@@ -36,19 +36,30 @@ export const handle: BreadcrumbHandle<typeof loader> = {
 }
 
 export async function loader({
-  params: { circularId, version },
+  params: { circularId, version, next, previous },
```

Comment:
These params appear to be unused.

```suggestion
  params: { circularId, version },
```

Reply:
oops, yeah, neglected to remove those from an earlier iteration